firmware/controllers/map_averaging.cpp

```diff
@@ -240,7 +269,7 @@ float getMap(void) {
 #if EFI_ANALOG_SENSORS || defined(__DOXYGEN__)
 	if (!isValidRpm(engine->rpmCalculator.rpmValue))
 		return validateMap(getRawMap()); // maybe return NaN in case of stopped engine?
-	return validateMap(getMapByVoltage(v_averagedMapValue));
+	return validateMap(minMapPressure);
```

Comment:
here 'minMapPressure' name is really just too much information, and in reality it;s 'min averaged' pressure

how about we rename to 'currentPressure' and add a one-line comment next to the declaration explaining that this is 'minimal averaged' MAP?